scripts/proof-run-analyze.py

```
@@ -0,0 +1,610 @@
+                time_vals.append(m["wall_clock_ms"])
+
+        sys_name = SYSTEMS.get(q["system"], q["system"])
+
+        if n_runs > 1 and token_vals:
+            tq1, tmed, tq3 = compute_iqr(token_vals)
+            wq1, wmed, wq3 = (
+                compute_iqr(time_vals) if time_vals else (0, 0, 0)
+            )
+            lines.append(
+                f"| {qid} | {sys_name} | {q['type']} | "
+                f"{tmed:,.0f} | {tq1:,.0f}–{tq3:,.0f} | "
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)